orcestra/flightplan.py

```diff
@@ -45,6 +47,17 @@ class LatLon:
     lon: float
     label: Optional[str] = None
     fl: Optional[float] = None
+    time: Optional[datetime.datetime | str] = None
```

Comment:
Currently, `parse_datestr()` only accepts strings and will crash for datetime objects. This was intentional as it copies the behaviour of `datetime.fromisoformat()`. However, I am fine with relaxing this